Desktop Notifier/script.py

```diff
@@ -0,0 +1,25 @@
+import time
+from plyer import notification
+import requests
+import json
+
+
+
+news = requests.get('https://newsapi.org/v2/top-headlines?country=in&apiKey=51606761f68a4b20b4296e61814dbd1a')
```

Comment:
take input for api key, and country from the user before notifying

Reply:
@kaustubhgupta  sir how to write the variable name in the url?? 
Please help me with it.

Reply:
as a URL is string itself, you can use the concept of f-strings. Example:
```python
video_id = "bqOfdv9i4sE"
URL = f"https://www.youtube.com/watch?v={video_id}"
```